AppendBlobReader/src/appendblobreaderdeploy.json

```diff
@@ -190,17 +187,15 @@
                                     "isSubjectCaseSensitive": false,
                                     "includedEventTypes": [
                                         "Microsoft.Storage.BlobCreated"
-                                    ],
-                                    "enableAdvancedFilteringOnArrays": true
+                                    ]
                                 },
-                                "eventDeliverySchema": "EventGridSchema",
```

Comment:
eventDeliverySchema should be present as was in the older one
same with enableAdvancedFilteringOnArrays